tests/python/tests/http_server/php/index.php

```diff
@@ -8,6 +8,55 @@ class A {
   public $b = "hello";
 }
 
+/**
+ * @kphp-required
+ */
+function shutdown_function() {
+    fwrite(STDERR, "shutdown_function was called\n");
+}
+
+interface I {
+    public function work();
+}
+
+class ResumableWorker implements I {
+    public function work() {
+        $job = function($x) {
+                sched_yield_sleep(2);
+                return $x;
+        };
+        $futures = [];
+        for ($i = 0; $i < 10; ++$i) {
+            $futures[] = fork($job($i));
+        }
+        wait_multi($futures);
+        fwrite(STDERR, "test_ignore_user_abort/finish_resumable_work\n");
+    }
+}
+
+class RpcWorker implements I {
+    protected int $port;
+
+    public function __construct(int $port) {
+        $this->port = $port;
+    }
+
+    public function work() {
+        fwrite(STDERR, $this->port . "\n");
+        $job = function() {
+            $conn = new_rpc_connection('localhost', $this->port, 0, 3);
+            $req_id = rpc_tl_query_one($conn, ["_" => "engine.sleep",
+                                                "time_ms" => 60]);
+            $resp = rpc_tl_query_result_one($req_id);
```

Comment:
let's add an assertion that response is TL Bool true

Reply:
fixed

Reply:
fix